KsApi/models/graphql/GraphCommentsEnvelope.swift

```diff
@@ -24,12 +27,19 @@ extension GraphCommentsEnvelope {
 
   init(from decoder: Decoder) throws {
     let values = try decoder.container(keyedBy: CodingKeys.self)
+    let container: KeyedDecodingContainer<GraphCommentsEnvelope.CodingKeys>
```

Comment:
can this implementation be done without declaring this constant here? I think you can avoid it?

Reply:
I don't think so because you can't use `try?` with nil-coalescing.

Reply:
I was just curious if this was actually possible and looks like it can be done this way:

```swift
/// Try to decode a `Project` then `Update` container.
let tryContainer = (
  (try? values.nestedContainer(keyedBy: CodingKeys.self, forKey: .project)) ??
  (try? values.nestedContainer(keyedBy: CodingKeys.self, forKey: .post))
)

guard let container = tryContainer else {
  throw DecodingError.dataCorruptedError(
    forKey: .post,
    in: values,
    debugDescription: "Unable to decode a Project or Update."
  )
}

self.slug = try container.decodeIfPresent(String.self, forKey: .slug)
self.updateID = try container.decodeIfPresent(String.self, forKey: .id)
```

I had hoped you could coalesce from `try?` to `try` but Swift doesn't allow you to put a `try` on the right-hand side of the `??` operator.